.github/workflows/dependabot.yaml

```
@@ -0,0 +1,92 @@
+# https://docs.github.com/en/actions
+
+name: "Dependabot Merge"
+
+on:
```

```suggestion
on: # yamllint disable-line rule:truthy
```

Will fix the error from yamllint.

---

thanks. i see that in the other workflows now.